Makefile

```diff
@@ -39,8 +39,13 @@ build-test-container:
 test: build-test-container test-in-container test-doc-examples
 
 test-in-container:
-	@# use it like this: `make test-in-container TEST_TARGET=tests/integration/test_utils.py`
-	docker run --net=host --rm -v /dev:/dev:ro -v /var/lib/docker:/var/lib/docker:ro --security-opt label=disable --cap-add SYS_ADMIN -ti -v /var/run/docker.sock:/var/run/docker.sock -v $(CURDIR):/src -v $(CURDIR)/pytest-container.ini:/src/pytest.ini $(TEST_IMAGE_NAME) /bin/bash -c "oc login 127.0.0.1:8443 -u developer -p developer -n conu --insecure-skip-tls-verify && make exec-test TEST_TARGET=$(TEST_TARGET)"
+	@# use it like this: `make test-in-container TEST_TARGET=tests/integration/test_utils.py`k
+	docker run --net=host --rm -v /dev:/dev:ro -v /var/lib/docker:/var/lib/docker:ro --security-opt label=disable --cap-add SYS_ADMIN -e KUBECONFIG=/var/lib/origin/openshift.local.config/master/admin.kubeconfig -ti -v /var/run/docker.sock:/var/run/docker.sock -v $(CURDIR):/src -v $(CURDIR)/pytest-container.ini:/src/pytest.ini $(TEST_IMAGE_NAME) /bin/bash -c "oc login 127.0.0.1:8443 -u developer -p developer -n conu --insecure-skip-tls-verify && make exec-test TEST_TARGET=$(TEST_TARGET)"
+
+test-k8s-minikube: build-test-container
```

Comment:
Shouldn't the `test-in-container` target also have `build-test-container` as dependency ?